### PR TITLE
Contact form

### PR DESCRIPTION
**Pull request by m05e5**
HTML changes: created the footer with the contact form
CSS changes: styled the footer and the contact form
organized my assets

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -211,96 +208,111 @@ <h1 class="card-title">Myself</h1>
...
+    </section>
+    <footer id="contact-me">
+        <h3 id=contact-head>Contact me</h3>
+        <p id="contact-description">
+            If you have an application you are interested 
+            in developing, a feature that you need built 
+            or a project that needs coding. I’d love to 
+            help with it 
+        </p>
+        <form id="form" action="#">
```
Comment (Dipeshtwis) [inline index.html]: - [ ] Form uses the action and method attribute to integrate with [formspree service](https://formspree.io/html). (Register here and use your own form_id)
- action value should be similar to https://formspree.io/f/{form_id}.
- method value equals POST (`method = "POST"`)

File: works.css [hunk 2]
```diff
@@ -161,6 +163,75 @@ h1 {
...
+  border-radius: 9px;
+  border: 1px solid rgb(206, 218, 216);
+}
+
+#message {
+  width: 100%;
+  margin-bottom: 1rem;
+  border-radius: 9px;
+  border: 1px solid rgb(206, 218, 216);
+}
```
Comment (Dipeshtwis) [inline works.css]: - [ ] Always use the class attribute for multiple elements (do not use ID selector for multiple elements).
- [ ] Here you are not following the DRY principle. name, email, and message has same property, you can avoid repeating this using class and use that class in HTML wherever you need.

File: works.css [hunk 3]
```diff
@@ -161,6 +163,75 @@ h1 {
...
+  background-position: initial;
+}
+
+#contact-head {
+  margin-top: 8rem;
+  font-weight: bold;
+  font-size: 40px;
+}
+
+#contact-description {
```
Comment (Dipeshtwis) [inline works.css]: [OPTIONAL]

- Avoid using `Id` much.

**Review by Dipeshtwis — CHANGES_REQUESTED** [review]
Hi @m05e5,

:+1: This is good to make it this far, your project is really nice, congratulations on the good work you have done so far.
:+1: Good use of Flexbox.
:+1:  PR title and description is good.

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!


 ## Required Changes ♻️

- [ ] README file should have description about the project even if it's one or two line so that anyone can understand about your project. Also there is broken screenshot link in your README file. Please fix that.


**_Also Check the comment under the review_**


## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

**_Check the comment under the review_**

Cheers and Happy coding!👏👏👏
Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

**_Dipesh Kumar_**


------

_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have only 3 code reviews per project. If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: index.html (hunk 1 above)
Comment (DeliceLydia) [reply index.html]: Well addressed the issues!

File: works.css (hunk 2 above)
Comment (DeliceLydia) [reply works.css]: Resolved this issue!

File: works.css (hunk 3 above)
Comment (DeliceLydia) [reply works.css]: I can see that you reduced the use of ID's. well done!

**Review by DeliceLydia — APPROVED** [review]
# Status: Approved 🎉 🎉 

Hi @m05e5, 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 



### Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.


------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have only 3 code reviews per project. If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**